### PR TITLE
IGVF-450-phenotypic-feature

### PR DESCRIPTION
**Pull request by mingjiecn**

### CODE REVIEW

File: lib/phenotypic-feature.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Use the same title for biomarker search page and biomarker object page for that biomarker
+export function getPhenotypicFeatureTitle(phenotypicFeature) {
+  const title = phenotypicFeature.quantity
+    ? phenotypicFeature.feature.term_name +
+      " - " +
+      phenotypicFeature.quantity +
+      " " +
+      phenotypicFeature.quantity_units
+    : phenotypicFeature.feature.term_name;
+  return title;
```
Comment (forresttanaka) [inline lib/phenotypic-feature.js:10]: I think using a string template would make this easier to read and maintain.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
😎
Just a couple small changes, plus reverting the impersonation menu removal.

File: components/navigation.js [hunk 2]
```diff
@@ -870,17 +881,7 @@ function NavigationExpanded({ navigationClick, toggleNavCollapsed }) {
...
-                id="impersonate"
-                href="/impersonate-user"
-                navigationClick={navigationClick}
-                isChildItem
-              >
-                Impersonate User
-              </NavigationHrefItem>
-            )}
-            <NavigationSignOutItem id="signout" isChildItem>
+            <NavigationSignOutItem id="sign-out" isChildItem>
```
[the remark this answers is not in the document]
Comment (mingjiecn) [reply components/navigation.js:884]: I have no idea what happened here...I don't recall I changed anything here... So strange!

File: lib/phenotypic-feature.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// Use the same title for biomarker search page and biomarker object page for that biomarker
```
[the remark this answers is not in the document]
Comment (mingjiecn) [reply lib/phenotypic-feature.js:1]: I think I had some commit issue and some of my code got eaten...

**Review by forresttanaka — CHANGES_REQUESTED** [review]
Awesome! I think one check might not be correct.

File: pages/phenotypic-features/[id].js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+              {phenotypicFeature.quantity && (
+                <>
+                  <DataItemLabel>Quantity</DataItemLabel>
+                  <DataItemValue>
+                    {phenotypicFeature.quantity}{" "}
+                    {phenotypicFeature.quantity_units}
+                  </DataItemValue>
+                </>
+              )}
+              {phenotypicFeature.quantity && (
```
[the remark this answers is not in the document]
Comment (mingjiecn) [reply pages/phenotypic-features/[id].js:52]: oops...

**Review by forresttanaka — APPROVED** [review]
👍

**Comment by forresttanaka** [comment]
@mingjiecn Before you start a demo, clear the out-of-date warning by rebasing on dev.

**Pull request merged**